repositories/rosbag2.BUILD.bazel

```diff
@@ -81,7 +132,9 @@ ros2_cpp_library(
         "rosbag2_cpp/src/rosbag2_cpp/reindexer.cpp",
     ],
     hdrs = glob(["rosbag2_cpp/include/**/*.hpp"]),
-    data = [":rosbag2_storage_default_plugins"],
+    data = [
+        ":rosbag2_storage_plugins",
```

Comment:
Or we can opt here to put here more plugins.

Reply:
The problem was, that if I add both plugins separately here, two plugins.xml files are being written which conflict.

Reply:
I fixed this issue in #69 , you can split the plugins now.